CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-project(octotiger C CXX)
+project(octotiger C CXX CUDA)
```

Comment:
Neither CMake 2.8 nor 3.3.2 support this. First class support for CUDA is introduced in CMake 3.10. Either raise the minimum required to 3.10 or remove this.

Reference:
* [FindCUDA - CMake 3.10](https://cmake.org/cmake/help/v3.10/module/FindCUDA.html)
* [FindCUDA - CMake 3.9](https://cmake.org/cmake/help/v3.9/module/FindCUDA.html)
* <https://cliutils.gitlab.io/modern-cmake/chapters/packages/CUDA.html>